.gitignore

```diff
@@ -15,6 +15,7 @@ requests.egg-info/
 *.swp
 *.egg
 env/
+.idea/
```

Comment:
It seemed like the right thing to do since there was no CONTRIBUTING document explaining what the rules were and this seemed like the most straightforward way to avoid getting the project files into the code both for me and future contributors who happen to use PyCharm / IDEA. 

You recommend I open a separate issue just for this one-line change to .gitignore, then? If that's the right thing to do, I'll do it.